pax/plugins/signal_processing/PulseShapeDiscrimination.py

```diff
@@ -0,0 +1,146 @@
+import numpy
+import itertools
+
+from pax import plugin, dsputils
+
+class F90(plugin.TransformPlugin):
+    def startup(self):
+        self.dt = self.config['sample_duration'] # Some variable from config
+        self.reference_baseline = self.config['digitizer_reference_baseline']
+
+    def transform_event(self, event):
+        found_pulse = set()
+        for s1 in event.s1s():
+            fderivative = s1.sum_waveform[2:] - s1.sum_waveform[:-2]
+            sderivative = s1.sum_waveform[2:] + s1.sum_waveform[:-2] - 2 * s1.sum_waveform[1:-1]
+            try:
+                starttime = [index for index, (slope, curvature) in enumerate(zip(fderivative, sderivative)) if (slope > 0.20 and curvature > 0.20)][0] + 1
+                print(starttime)
+            except IndexError:
+                continue
+            f10_int = 0.0
+            f20_int = 0.0
+            f30_int = 0.0
+       	    f40_int = 0.0
+            f50_int = 0.0
+       	    f60_int = 0.0
+            f70_int = 0.0
+       	    f80_int = 0.0
+            f90_int = 0.0
+            f100_int = 0.0
+       	    f110_int = 0.0
+            f120_int = 0.0
+            f130_int = 0.0
+            f140_int = 0.0
+            f150_int = 0.0
+            f160_int = 0.0
+            f170_int = 0.0
+            f180_int = 0.0
+       	    f190_int = 0.0
+            f200_int = 0.0
+            total_int = 0.0
+            s1right = s1.right
+            for index, hit in enumerate(s1.hits):
+                if(hit['found_in_pulse'] not in found_pulse):
+                    pulse = event.pulses[hit['found_in_pulse']]
+                    found_pulse.add(hit['found_in_pulse'])
+                    time = [t for t in range(pulse.left, pulse.right + 1)]
```

Comment:
__[flake8]__
* [F841] local variable 'time' is assigned to but never used

<!-- Comment by [SideCI](https://sideci.com) -->